libcheckisomd5.c

```diff
@@ -21,7 +21,7 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
  */
 
-
+#include <malloc.h>
```

Comment:
Because `memalign` is provided by `malloc.h` on most systems. But looking at this again it seems like `alligned_alloc` is probably the better choice since it's even c11 standard.